src/SendGrid/Helpers/Errors/Model/RequestErrorException.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace SendGrid.Helpers.Errors.Model
+{
+    /// <summary>
+    /// Represents errors with status code 4xx. Being x any number
+    /// </summary>
+    public class RequestErrorException : Exception
```

Comment:
Not sure I understand why we have custom exceptions for 403, 404, 405, 413, etc. as well as a 4xx catch-all exception. Is this just for more granularity on commonly thrown 4xx exceptions?

Reply:
Yes, that's correct.

I'm making a change now so the more specific 4xx and 5xx exceptions inherit from these bases.